Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -74,7 +77,9 @@ enum BacktraceBreadcrumbFileHelperError: Error {
 
     func clear() -> Bool {
         do {
-            try queue.clearAndReturnError()
+            try dispatchQueue.sync {
+                try queue.clearAndReturnError()
```

Comment:
clearAndReturnError returns boolean - maybe it's better to use boolean instead?

Reply:
This is part of the ObjC/Swift magic. The boolean is not exposed.

```
Objective-C methods that return BOOL and return an NSError by reference are exposed to Swift as methods that use the try/catch error-handling mechanism, and return Void.
```

https://stackoverflow.com/questions/46626571/how-to-convert-objective-c-bool-to-swifts-bool-with-nserror-as-one-of-the-par